lib/helpers/cursor/eachAsync.js

```diff
@@ -62,14 +64,19 @@ module.exports = function eachAsync(next, fn, options, callback) {
           return done();
         }
         if (err != null) {
-          error = err;
-          finalCallback(err);
-          return done();
+          if (continueOnError) {
+            aggregatedErrors = aggregatedErrors || new EachAsyncMultiError([]);
```

Comment:
`aggregatedErrors` is a bit confusing, it makes me think that it's an array of errors.
Perhaps `eachMultiAsyncError` might be a more descriptive name.